PhysicsTools/PatUtils/interface/MiniIsolation.h

```diff
@@ -0,0 +1,35 @@
+#ifndef MiniIsolation_h
+#define MiniIsolation_h
```

Comment:
Please use an include guard matching the full file name subsystem_package_fileName_h